bucket.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import random
+from pathlib import Path
+
+import boto3
+import botocore
+
+import dynaconfig
+
+session = boto3.session.Session()
+
+BUCKET_NAME = dynaconfig.settings["BUCKET_NAME"]
+ENDPOINT_URL = dynaconfig.settings["ENDPOINT_URL"]
+REGION_NAME = dynaconfig.settings["REGION_NAME"]
+AWS_ACCESS_KEY_ID = dynaconfig.settings["AWS_ACCESS_KEY_ID"]
+AWS_SECRET_ACCESS_KEY = dynaconfig.settings["AWS_SECRET_ACCESS_KEY"]
+
+CLIENT = session.client(
+    "s3",
+    endpoint_url=ENDPOINT_URL,
+    config=botocore.config.Config(s3={"addressing_style": "virtual"}),
+    region_name=REGION_NAME,
+    aws_access_key_id=AWS_ACCESS_KEY_ID,
+    aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+)
+
+# Logging
+logging.basicConfig(
+    format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO
+)
+logging.basicConfig(
+    format="%(asctime)s - %(levelname)s - %(message)s", level=logging.ERROR
+)
+
+
+def filter_conf(dir_to_filter):
```

Comment:
Add the return-annotation.

Reply:
Have no idea what the closure func return